includes/Core/Storage/Owner_ID.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Storage\Owner_ID
+ *
+ * @package   Google\Site_Kit\Core\Storage
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Storage;
+
+/**
+ * Owner_ID class.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Owner_ID extends Setting {
+
+	/**
+	 * The option_name for this setting.
+	 */
+	const OPTION = 'googlesitekit_owner_id';
+
+	/**
+	 * Gets the expected value type.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string The type name.
+	 */
+	protected function get_type() {
+		return 'integer';
+	}
+
+	/**
+	 * Gets the value of the setting.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return int Value set for the option, or registered default if not set.
+	 */
+	public function get() {
+		$owner_id = parent::get();
+		if ( ! $this->has() ) {
+			$owner_id = $this->get_default();
+		}
+
+		return intval( $owner_id );
```

Comment:
Couldn't we simplify this to:
```suggestion
		return (int) parent::get();
```

The default should be handled by WordPress already, via `register_setting`.

Reply:
Actually, we don't need to override `get` at all. Removed.